src/Simplex/Messaging/Agent/Store/SQLite.hs

```diff
@@ -45,10 +46,15 @@ data SQLiteStore = SQLiteStore
     dbConn :: DB.Connection
   }
 
-newSQLiteStore :: MonadUnliftIO m => String -> m SQLiteStore
-newSQLiteStore dbFilename = do
+createSQLiteStore :: MonadUnliftIO m => String -> m SQLiteStore
+createSQLiteStore dbFilename = do
+  store <- connectSQLiteStore dbFilename
```

Comment:
add the following TODO?

`TODO check that embedded SQLite library is compiled with threadsafe code [and if not raise an exception?]; this can be achieved by running "pragma COMPILE_OPTIONS;" and checking "SQLITE_THREADSAFE=1" or "SQLITE_THREADSAFE=2" is in the list of compile options (and there shouldn't be "SQLITE_THREADSAFE=0")`

this may be unnecessary if later we switch to TBQueue for queries across all threads, but at least would be a reminder until then